config/yurtadm/cloudimage/README.md

```diff
@@ -0,0 +1,13 @@
+# Build an OpenYurt CloudImage
+
+`yurtadm init` is implemented by sealer, you can modify the kubefile to make your own openyurt cloudimage.
+
+```bash
+cd openyurt-latest
+
+# build cloudimage
+sealer build -t registry-1.docker.io/openyurt/openyurt-cluster:latest-k8s-1198 -f Kubefile .
+
+# push to dockerhub
+sealer push registry-1.docker.io/openyurt/openyurt-cluster:latest-k8s-1198
```

Comment:
i think that common end user can not push openyurt-cluster image into OpenYurt dockerhub, because they can not  get the password.

Reply:
OK. It is just an example.